static/js/sefaria/sefaria.js

```diff
@@ -1943,6 +1943,17 @@ _media: {},
           data["completions"][0].toLowerCase().replace('״','"') == query.slice(0, data["completions"][0].length).toLowerCase().replace('״','"') &&
           data["completions"][0] != query.slice(0, data["completions"][0].length))
   },
+  isGershayimVariant: function(query, data) {
+    function replace_gershayim_by_merkhaot(string) {
+        return string.replace('"', '״');
+    }
+    return (
+      !(data["is_ref"]) &&
+      data.completions &&
+      !data.completions.includes(query) &&
```

Comment:
previous line is redundant. this line can be `!data.completions?.includes(query)`